src/inflators/utils.js

```diff
@@ -30,3 +30,9 @@ export const createEntityAndReparent = (node, klass = undefined) => {
   sceneEl.object3D.attach(node);
   return entity;
 };
+
+export const absoluteURLForAsset =
+  window.absoluteURLForAsset ||
+  ((asset) => {
+    return asset;
+  });
```

Comment:
Can you please remove it from inflators/environment-settings.js and import it from here as well.

Reply:
Done.